### PR TITLE
Editor select

### LINKED ISSUE #42: Suggested optional config files for editors?
It could be nice to let the user select their editor during configuration and get a file similar to the .vscode config, but for other editors such as neovim or helix

This is what helix would look like for example.
`.helix/languages.toml`
```toml
[[language]]
name = "rust"

[language-server.rust-analyzer.config]
check.allTargets = false
cargo.target = "riscv32imac-unknown-none-elf"
```

### PR DESCRIPTION
**Pull request by rmsthebest**
Mostly fixes #42

I will admit I don't know how to configure neovim, but I've added support for the editor I use anyway.
I could have added editors under the `optional` section but i felt as if having deeper nested selection was a worse experience. Can move it if you want me to

I snuck in space to select as well as that is quite a common key in these terminal interfaces.

### CODE REVIEW

File: template/.helix/languages.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#INCLUDEFILE helix
+[[language]]
+name = "rust"
+
+[language-server.rust-analyzer.config]
+check.allTargets = false
+cargo.target = "riscv32imac-unknown-none-elf"
```
Comment (SergioGasquez) [inline template/.helix/languages.toml:7]:
```suggestion
check.allTargets = false
#REPLACE riscv32imac-unknown-none-elf rust_target
cargo.target = "riscv32imac-unknown-none-elf"
```

**Review by SergioGasquez — COMMENTED** [review]
Thanks for the contribution! I think we need to update the target on the helix config file and update the changelog file

File: template/.helix/languages.toml (hunk 1 above)
Comment (rmsthebest) [reply template/.helix/languages.toml]: Right, I saw that in the vscode one but for some reason I forgot to think about what it did

**Review by jessebraham — APPROVED** [review]
Thanks for the PR! I had done this locally and forgot to open a PR myself 😅 But this appears to match what I did almost exactly so LGTM!

**Pull request merged**